cmd/main.go

```diff
@@ -455,6 +473,104 @@ func run(mainConfig Config) (err error) {
 	return nil
 }
 
+func staticRun(mainConfig Config) (err error) {
+	trustDomain, err := spiffeid.TrustDomainFromString(mainConfig.ctrlConfig.TrustDomain)
+	if err != nil {
+		setupLog.Error(err, "invalid trust domain name")
+		return err
+	}
+
+	ctx := ctrl.SetupSignalHandler()
+
+	setupLog.Info("Dialing SPIRE Server socket")
+	spireClient, err := spireapi.DialSocket(mainConfig.ctrlConfig.SPIREServerSocketPath)
+	if err != nil {
+		setupLog.Error(err, "unable to dial SPIRE Server socket")
+		return err
+	}
+	defer spireClient.Close()
+
+	var mgr ctrl.Manager
+	if mainConfig.ctrlConfig.StaticManifestPath != nil {
+		mgr, err = ctrl.NewManager(&rest.Config{}, mainConfig.options)
+	} else {
+		mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), mainConfig.options)
```

Comment:
this is never going to be true right?
we are only going to call static when `StaticManifestPath != nil`,
if that is the case may we fail here?

Reply:
oh. left over from before. will fix.